tests/cypress/support/commands.js

```diff
@@ -312,6 +312,9 @@ Cypress.Commands.add('createPoint', (createPointParams) => {
     createPointParams.pointsMap.forEach((element) => {
         cy.get('.cvat-canvas-container').click(element.x, element.y);
     });
+    if (createPointParams.useDoneButton) {
+        cy.contains('span', 'Done').click();
+    }
```

Comment:
better to use else with the following condition, because both useDoneButton, complete cannot be used together

Reply:
Reworked.